libr/util/str.c

```diff
@@ -216,7 +216,7 @@ R_API int r_str_binstr2bin(const char *str, ut8 *out, int outlen) {
 // Returns the permissions as in integer given an input in the form of rwx, rx,
 // etc.
 R_API int r_str_rwx(const char *str) {
-	int ret = atoi (str);
```

Comment:
I'm not sure we should rely on this kind of behaviour. So, how does one know, without reading the code, that 17 is "m--x"?